File: case-study/src/app/model/attach-service.ts
export class AttachService {
  private _id: number;
  private _name: string;
  private _unit: string;
  private _unitPrice: number;


  constructor(id: number, name: string, unit: string, unitPrice: number) {
    this._id = id;
    this._name = name;
    this._unit = unit;
    this._unitPrice = unitPrice;
  }


  get id(): number {
    return this._id;
  }

  set id(value: number) {
    this._id = value;
  }

  get name(): string {
    return this._name;
  }

  set name(value: string) {
    this._name = value;
  }

  get unit(): string {
    return this._unit;
  }

  set unit(value: string) {
    this._unit = value;
  }

  get unitPrice(): number {
    return this._unitPrice;
  }

  set unitPrice(value: number) {
    this._unitPrice = value;
  }
}

File: back-end/src/main/java/com/codegym/model/AttachService.java
package com.codegym.model;

import javax.persistence.*;
import java.util.List;

@Entity
public class AttachService {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;
    private String name;
    private String unit;
    private Long unitPrice;
    @OneToMany(mappedBy = "attachService")
    private List<Service> serviceList;

    public AttachService() {
    }

    public AttachService(Long id, String name, String unit, Long unitPrice, List<Service> serviceList) {
        this.id = id;
        this.name = name;
        this.unit = unit;
        this.unitPrice = unitPrice;
        this.serviceList = serviceList;
    }

    public Long getId() {
        return id;
    }

    public void setId(Long id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getUnit() {
        return unit;
    }

    public void setUnit(String unit) {
        this.unit = unit;
    }

    public Long getUnitPrice() {
        return unitPrice;
    }

    public void setUnitPrice(Long unitPrice) {
        this.unitPrice = unitPrice;
    }

    public List<Service> getService() {
        return serviceList;
    }

    public void setService(List<Service> service) {
        this.serviceList = service;
    }
}
